layers/+lang/python/packages.el

```diff
@@ -100,8 +100,7 @@
   (use-package code-cells
     :if (not (configuration-layer/layer-used-p 'ipython-notebook))
     :defer t
-    :commands (code-cells-mode)
-    :init
+    :config
     (progn
       (add-hook 'python-mode-hook 'code-cells-mode)
```

Comment:
moving this line to :config is problematic. it won't be added to the hook until it's loaded, which means you need to manually load it.

Reply:
But having on :init causes `describe-key` to break ... :/

The issue seems to be that `(symbol-value 'code-cells-mode)` does not triggers an auto-load for code-cells

For instance: this fails in the `(message (symbol-value mode))` statement
```
(mapcar
                                       (lambda (mode-and-map)
                                         (message "mode and map: %S" mode-and-map)
                                         (let ((mode (car mode-and-map)))
                                           (message "trying to get value for %S" mode)
                                           (message "symbol-value: %S" (symbol-value mode))
                                           (when (symbol-value mode)
                                             (intern-soft
                                              (format "%s-map" mode)))))
                                       minor-mode-map-alist)
```

Reply:
what I really don't understand is that code-cells-mode is already autoloaded at current version

https://github.com/astoff/code-cells.el/blob/fd68a33eb43b3cbd44fed767f48e230382903592/code-cells.el#L303

what if we remove the `:command (code-cells-mode)` and keep other lines unchanged?